src/gui/qjacktrip.cpp

```diff
@@ -108,7 +100,11 @@ QJackTrip::QJackTrip(int argc, bool suppressCommandlineWarning, QWidget* parent)
         About about(this);
         about.exec();
     });
-#ifndef NO_VS
+#ifdef NO_VS
+    m_ui->authNotVSLabel->setText(QStringLiteral(
+        "(This is for JackTrip's inbuilt authentication system. To easily "
```

Comment:
Suggestion:
`This is for JackTrip's inbuilt authentication system.` -> `This is for JackTrip's inbuilt authentication system, for the Virtual Studio.`

Reply:
Then we'd have it three times on a build where it's disabled, the installation of which was probably a conscious choice.

Reply:
I'm just wary of the explanatory text taking up more vertical screen space than the actual important, actionable parts of that group box.

Reply:
That said maybe replacing 'build' with 'version of the program' is a little more layperson friendly?

Reply:
Good point - I was looking at the Virtual Studio build.